### PR TITLE
feat(drivers): nearly completed SR830 properties and method at stanford830@v1.0.0

### PR DESCRIPTION
**Pull request by rsbrost**
The stanford and keithley notebooks may be incomplete; however, the demo drivers test nb is sufficient for now, there are fundamentally important updates to the auto_test_driver.py file for testing drivers with read_only/write_only properties as well as to generate test logs which will be important for all drivers. This is a fairly massive update and for these reasons I suggest we go over it meticulously and pull before letting it get much larger.

### CODE REVIEW

File: pyscan/drivers/testing/test_instrument_driver.py
Comment (i-am-mounce) [inline pyscan/drivers/testing/test_instrument_driver.py:1]: remove the @property of version.

File: pyscan/general/get_version.py
Comment (i-am-mounce) [inline pyscan/general/get_version.py:1]: Is this get_version of pyscan or get_version of the drivers? Be more specific.

**Review by i-am-mounce — COMMENTED** [review]
Make changes suggested here and if the test_driver notebooks run, I think we can merge

**Comment by ammounce** [comment]
Purge the test logs. Pull the version from main.

**Review by i-am-mounce — APPROVED** [review]

**Comment by i-am-mounce** [comment]
We need tags for the conventional change log, also you have conflicts.

**Review by i-am-mounce — CHANGES_REQUESTED** [review]
Must have conventional change log tags and resolve conflicts.

**Review by i-am-mounce — APPROVED** [review]

**Pull request merged**